### ISSUE TITLE
questions about training and export onnx 

### ISSUE THREAD

**Issue by Sun-Pak** [eriklindernoren/PyTorch-YOLOv3#707]
Hi,
I'm trying to train custom dataset in google colab.

i got 2 questions.

First, 
when i detected my sample data, i didn't get desired result.
(The purpose is to capture 3 classes specified in one image )

I only trained 355 images. train image 277, valid image 78.

is too little? 

Last.
i don't know how to  export onnx.
when i trained pycham( local computer), added save onnx function end of run() in train.py

<img width="436" alt="run" src="https://user-images.githubusercontent.com/16331744/123887365-55056d80-d98c-11eb-9ffa-c3978b24a01d.PNG">

And this is my save code.
<img width="643" alt="save" src="https://user-images.githubusercontent.com/16331744/123887398-62225c80-d98c-11eb-809e-d944d8bae214.PNG">

In pycham, after train, it didn't export.

what did i do wrong

**Comment by Flova** [comment]
Depending on the task it seems like too little training data.

I would not add the onnx export to the end of the training, but make a simple script that loads the model as in the example in the `README.md` and does conversion afterward, so you don't need to finish the training before converting your stuff.
Also, it seems like you got the dynamic axis for onnx wrong. This is only needed if you want to do inference using multiple batch sizes and then that would be the batch dimension.

**Comment by Sun-Pak** [comment]
yes, i tried the example in the README.md it did well.
well.. What I ultimately want is to output as an onnx model.
As you say, i deleted dynamic_axes, but couldn't export onnx.

some errors occurred.
<img width="1338" alt="2233" src="https://user-images.githubusercontent.com/16331744/124102400-4ce82380-da9b-11eb-9d38-ea66b9b0b2e7.PNG">

**Comment by Flova** [comment]
These messages aren't errors but warnings. The first one is okay as long as your input resolution doesn't change for the inference. The second one is related to YOLOv4 but it should also be fine if these values are assumed static.
